norse/torch/functional/lif.py

```diff
@@ -465,13 +413,28 @@ def lif_feed_forward_step_sparse(
     i_decayed = state.i + di
 
     # compute new spikes
-    z_new = threshold(v_decayed - p.v_th, p.method, p.alpha)
```

Comment:
this shouldn't be mixed in here either ...

Reply:
Probably not :)